### ISSUE TITLE
Error when decoding an image using the 0.16.7 version

### ISSUE THREAD

**Issue by YeskaNova** [micjahn/ZXing.Net#402]
Hi,

I have an image in one of my unit tests that worked very well before I update to the 0.16.7 version. it throws An unhandled exception of type 'System.IndexOutOfRangeException' occurred in zxing.dll
Index was outside the bounds of the array. 

```using System.Drawing;
using ZXing;
using ZXing.Windows.Compatibility;

var reader = new BarcodeReader();
var result = reader.DecodeMultiple(new Bitmap(Bitmap.FromFile("e:\\test.bmp")));
```

I only reference ZXing.Net.Bindings.Windows.Compatibility version 0.16.10. And I'm targeting .NET 7 ( I had the same issue with .NET 6 )

The image is large enough, and clear enough.

[test.zip](https://github.com/micjahn/ZXing.Net/files/8010265/test.zip)

Thank you,

**Comment by micjahn** [comment]
It's a duplicate of #400
It will be fixed with 0.16.8. Working on a new release right now.
Workaround: activate every barcode type but Maxicode:
reader.Options.PossibleFormats = ...

**Comment by aliyailina** [comment]
@micjahn the same issue with QR_CODE format enabled.

StackTrace:

```
BitArray.get_Item (System.Int32 i)
RSSExpandedReader.getNextSecondBar (ZXing.Common.BitArray row, System.Int32 initialPos)
RSSExpandedReader.retrieveNextPair (ZXing.Common.BitArray row, System.Collections.Generic.List`1[T] previousPairs, System.Int32 rowNumber)
RSSExpandedReader.decodeRow2pairs (System.Int32 rowNumber, ZXing.Common.BitArray row)
RSSExpandedReader.decodeRow (System.Int32 rowNumber, ZXing.Common.BitArray row, System.Collections.Generic.IDictionary`2[TKey,TValue] hints)
MultiFormatOneDReader.decodeRow (System.Int32 rowNumber, ZXing.Common.BitArray row, System.Collections.Generic.IDictionary`2[TKey,TValue] hints)
OneDReader.doDecode (ZXing.BinaryBitmap image, System.Collections.Generic.IDictionary`2[TKey,TValue] hints)
OneDReader.decode (ZXing.BinaryBitmap image, System.Collections.Generic.IDictionary`2[TKey,TValue] hints)
MultiFormatReader.decodeInternal (ZXing.BinaryBitmap image, ZXing.ResultPointCallback rpCallback)
MultiFormatReader.decodeInternal (ZXing.BinaryBitmap image)
MultiFormatReader.decodeWithState (ZXing.BinaryBitmap image)
BarcodeReaderGeneric.Decode (ZXing.LuminanceSource luminanceSource)
BarcodeReader`1[T].Decode (T barcodeBitmap)
```
